### DISCUSSION TITLE
Simultaneously correctly/incorrectly recognized values

### DISCUSSION THREAD

**Discussion by mrmartin0815** [jomjol/AI-on-the-edge-device#2614] [General] [answered]
The "recognized digit ROIs" deviate from the detected/saved value. 
This causes the water meter to spontaneously deviate by 1qm.

![Water](https://github.com/jomjol/AI-on-the-edge-device-docs/assets/27679488/b5b362bf-2d85-4e39-bf07-e15132507a24)

**Comment by Th3H0ff** [comment, marked as answer]
I came here with the exact same issue. Below is my watermeter which is interpreted as 2805.9714 when it's in transition and should read 2804.9714. I've tried a myriad of different combinations of digital models and transition settings but so far no luck. What makes this extra annoying is having to wait for another 1000L of water to be expended before one can try another setting.
![Screenshot_20230928_192828_Chrome](https://github.com/jomjol/AI-on-the-edge-device/assets/86038326/573081a8-2bac-4251-a209-7d48ea6b747a)

Apart from this I love the project!


Update: I thought I'd give this another try and after alot of showering, dishes and clothes washing I think I've found a sweet spot that works for me. dig-cont_0620_s3_q and an analog/digital transition value of 8. Twice I've had my watermeter transition a while digit (1000L) without jumping ahead.
